### PR TITLE
Updating client image to lock it to ansible 2.3 and install some addi…

### PR DESCRIPTION
**Pull request by etsauer**
…tional dependencies

#### What does this PR do?
Adds some dependencies to the client image to account for new logging/metrics roles

#### How should this be manually tested?
Build the new client image, then run:

```
ansible-playbook -i /root/repository/casl-ansible-inventories/l1.casl.rht-labs.com.d/inventory/ repository/casl-ansible/playbooks/openshift/end-to-end.yml
```

#### Is there a relevant Issue open for this?
n/a

#### Who would you like to review this?
cc: @redhat-cop/casl

### CODE REVIEW

File: docker/openstack-client-centos/run.sh [hunk 1]
```diff
@@ -51,6 +51,9 @@ do
       echo "Warning: Setting the --host-net flag adds \`--net=host\` to your docker run command. Be aware that this means your container can talk to any network service running on this host."
       HOST_NET="--net=host"
       shift;;
+    --command=*)
```
Comment (oybed) [inline docker/openstack-client-centos/run.sh]: Can  you please update the usage above with the "command" option?

File: docker/openstack-client-centos/run.sh [hunk 2]
```diff
@@ -117,8 +120,12 @@ if [ -f ~/.ansible.cfg ]; then
...
+  docker run -it ${HOST_NET} ${REMOVE_CONTAINER_ON_EXIT} -v ${OPENSTACK_CONFIG_DIR}:/root/.config/openstack ${REPOSITORY_VOLUME} ${SSH_VOLUME} ${ANSIBLE_CFG} --entrypoint="${COMMAND}" ${OPENSTACK_CLIENT_IMAGE}
+else
 
-echo "Starting OpenStack Client Container...."
-echo
-#docker run -it ${HOST_NET} ${REMOVE_CONTAINER_ON_EXIT} -v ${OPENSTACK_CONFIG_DIR}:/root/.config/openstack:z ${REPOSITORY_VOLUME} ${SSH_VOLUME} ${ANSIBLE_CFG} ${OPENSTACK_CLIENT_IMAGE}
-docker run -it ${HOST_NET} ${REMOVE_CONTAINER_ON_EXIT} -v ${OPENSTACK_CONFIG_DIR}:/root/.config/openstack ${REPOSITORY_VOLUME} ${SSH_VOLUME} ${ANSIBLE_CFG} ${OPENSTACK_CLIENT_IMAGE}
+  echo "Starting OpenStack Client Container...."
+  echo
+  #docker run -it ${HOST_NET} ${REMOVE_CONTAINER_ON_EXIT} -v ${OPENSTACK_CONFIG_DIR}:/root/.config/openstack:z ${REPOSITORY_VOLUME} ${SSH_VOLUME} ${ANSIBLE_CFG} ${OPENSTACK_CLIENT_IMAGE}
```
Comment (oybed) [inline docker/openstack-client-centos/run.sh]: While you are at it, can we remove the commented out command that's no longer used?

**Comment by etsauer** [comment]
@oybed decided to go ahead and update this to address this: https://trello.com/c/Xps99uYm/235-bastionize-openstack-client-image

File: docker/openstack-client-centos/README.md [hunk 3]
```diff
@@ -20,45 +20,47 @@ systemctl enable docker
...
-
-     Usage: ./docker/openstack-docker-client/run.sh [options]
-     Options:
-     --configdir=<configdir>       : Directory containing Openstack configuration files (Default: ~/.openstack/)
-     --name=<name>                 : Name of the assembled image (Default: rhc-openstack-client)
-     --keep                        : Whether to keep the the container after exiting
-     --ssh=<ssh>                   : Location of SSH keys to mount into the container (Default: ~/.ssh)
-     --repository=<repository>     : Directory containing a repository to mount inside the container
-     --help                        : Show Usage Output
+cd ./docker/control-node-openstack
```
Comment (oybed) [inline docker/openstack-client-centos/README.md]: this doesn't look correct - shouldn't it be `docker/openstack-client-centos`? ... or did you mean to rename it to `control-node-openstack`?

Comment (oybed) [inline docker/openstack-client-centos/README.md]: @etsauer I'm fine with the rename. Go for it if you'd like.

**Comment by sabre1041** [comment]
@oybed can you approve and ill merge

**Pull request merged**